### DISCUSSION TITLE
Edges overlap with bezier edge type when source and target nodes are the same

### DISCUSSION THREAD

**Discussion by KumarilDave** [xyflow/xyflow#3548] [Q&A]
Hello friends,  when building a tree we have a use case when multiple edges have the same source and target nodes. I can understand that its not a normal tree structure but we want to want to be able to provide this as this is a common use case. From the documentation I see that using a customEdge as an option. The example is in the BiDirectionalEdge.tsx here https://reactflow.dev/docs/examples/edges/custom-edge/

I just wanted to see if there is any other better way to handle this. Thank you

**Comment by KriticaCelito** [comment]
Hi did you get the answer? I was looking for the same

**Comment by sinoryx** [comment]
Have the same feature request here!!!
